include/UniTensor.hpp

```diff
@@ -4165,6 +4165,26 @@ namespace cytnx {
       return UniTensor(cytnx::arange(Nelem), false, -1, in_labels, name);
     }
 
+    /**
+    @brief Generate a one-bond UniTensor with all elements are arange from 0 to Nelem-1, then
+    reshape to the given shape. Where Nelem is the product of the given shape.
+    @details Generate a UniTensor with all elements are arange from 0 to Nelem-1 with double data
+    type on cpu device. The step is 1.
+    @param[in] Nelem the number of elements.
+    @param[in] in_labels the labels of the UniTensor.
+    @param[in] name the name of the UniTensor.
+    @return
+        [UniTensor]
+    @see arange(const std::vector<cytnx_uint64> &shape, const cytnx_double &start, const
+    cytnx_double &end, const cytnx_double &step, const unsigned int &dtype, const int &device) \n
+    @see arange(const cytnx_int64 &Nelem)
+    */
+    static UniTensor arange(const std::vector<cytnx_uint64> &shape,
```

Comment:
same as above

Reply:
I think this should be kept?
We should remove arange(shape, start,end,...) ?

Reply:
you are combining arange and reshape at the same API call. numpy does not have this kind of APIs, and its inconsistent